src/lib/albertcore/src/albert/mainwindow/mainwindow.cpp

```diff
@@ -227,6 +227,12 @@ void MainWindow::setVisible(bool visible) {
 /** ***************************************************************************/
 void MainWindow::toggleVisibility() {
    setVisible(!isVisible());
+   // albert might fail to grab keyboard focus if you press keyboard immediately right after HOTKEY was triggered.
+   QTimer::singleShot(1, this, &MainWindow::regrabFocus);
```

Comment:
To your problem. Is the window visible when you are typing? The window has focus from the moment on it is visible. Maybe it takes some time on your system (which is an issue too).